modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -126,7 +129,7 @@
 				"mediaType": 0,
 				"iMediaID": this.getProxyConfig('MediaID'),
 				"iFileID": this.fileId,
-				"iLocation": this.getCurrentTime()
+				"iLocation": (this.getPlayer().isLive() && !this.getPlayer().isDVR())?0:this.getCurrentTime()
```

Comment:
Let's move this also to separate variables and use them here to shorten the condition.